src/main/java/hudson/plugins/gradle/injection/DevelocityExceptionLogProcessor.java

```diff
@@ -16,6 +16,14 @@ public final class DevelocityExceptionLogProcessor extends AbstractGradleLogProc
 
     private static final DevelocityExceptionDetector[] DETECTORS =
         {
+            new DevelocityExceptionDetector.ByPrefix(
+                BuildToolType.GRADLE,
+                "Internal error in Develocity Gradle plugin:"
+            ),
+            new DevelocityExceptionDetector.ByPrefix(
+                BuildToolType.MAVEN,
+                "[ERROR] Internal error in Develocity Maven extension:"
+            ),
             new DevelocityExceptionDetector.ByPrefix(
```

Comment:
❔ Why do we need this one? I think it is fine to always refer to the product as "Develocity".

Reply:
Old plugin versions will log the old string